### ISSUE TITLE
Recipient creation from within transaction form

### ISSUE THREAD

**Issue by nmorduch** [littleweaver/django-argus#24]
I think that this needs to be available from the transaction creation form. I would like both to use something like chosen for recipient selection (see #23) and to allow creation from the same field (see conversation on #15).

**Comment by melinath** [comment]
This would involve making recipient names unique within a group... which would make sense anyway.

The one issue would be - now that members and recipients are the same, which do we create? I'm guessing recipients.

**Comment by nmorduch** [comment]
Yeah, recipient I think. And then members are just a subset of the list of recipients?

**Comment by melinath** [comment]
Both are subsets of "parties"

**Comment by nmorduch** [comment]
oh right. i think it might make sense for members to be a subset of recipients, since all members are potential recipients. or are we thinking there might someday be other kinds of parties? birthday parties? costume parties?

**Comment by melinath** [comment]
eventually, there could be other types of "parties", like bank accounts, or sources of income.
